### DISCUSSION TITLE
iOS Live Photos

### DISCUSSION THREAD

**Discussion by konstantinmalich** [SixLabors/ImageSharp#1372] [Q&A]
Hello

I'm having an exception "Multiple SOF markers. Only single frame jpegs supported." when calling Image.Load() with iOS Live Photo.
Do you have any ideas how to avoid that?

**Comment by JimBobSquarePants** [comment]
Life photos aren't just jpegs. They also contain a Quicktime Mov instance.  

https://stackoverflow.com/questions/32508375/apple-live-photo-file-format
